build.sbt

```diff
@@ -3,19 +3,24 @@ name := "zeoflow"
 organization := "com.zeotap"
 
 scalaVersion := "2.11.12"
+version := "1.2"
 
 import ReleaseTransformations._
 
 val sparkVersion = "2.4.3"
 val beamVersion = "2.33.0"
 
+resolvers += "central" at "https://zeotap.jfrog.io/zeotap/libs-release"
+resolvers += "snapshot" at "https://zeotap.jfrog.io/zeotap/libs-snapshot"
+credentials += Credentials(new File(Path.userHome.absolutePath + "/.sbt/.credentials"))
+
 libraryDependencies ++= Seq(
   "com.fasterxml.jackson.module" % "jackson-module-paranamer" % "2.12.1",
   "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.12.1",
   "com.google.cloud.spark" %% "spark-bigquery-with-dependencies" % "0.21.1",
   "com.zeotap" %% "spark-property-tests" % "2.4.3",
   "com.zeotap" %% "data-expectations" % "1.2",
-  "com.zeotap" %% "data-io" % "1.1-SNAPSHOT",
+  "com.zeotap" %% "data-io" % "1.2.0" from "file:///Users/kushagra/.m2/repository/com/zeotap/data-io_2.11/1.2.0/data-io_2.11-1.2.0.jar",

```

Comment:
It will be changed once data-io PR is approved and relevant version gets released in JFrog